configs/webpack/base.coffee

```diff
@@ -10,7 +10,7 @@ _ = require 'lodash'
 # base config, true for all builds no matter what conditions
 BASE_CONFIG =
   cache: true
-  devtool: 'source-map'
+  devtool: 'cheap-module-eval-source-map'
```

Comment:
Does this still keep the minified JS file small? The only documentation I've seen on this is https://webpack.github.io/docs/configuration.html and it's sparse